torch/_inductor/mkldnn_lowerings.py

```diff
@@ -884,7 +884,7 @@ def qlinear_binary(
             x_size = x.get_size()
             x2_size = x2.get_size()
             assert len(x_size) == len(x2_size)
-            if len(x_size) > 2 and binary_attr == "add":
+            if len(x_size) > 2 and (binary_attr == "add" or binary_attr == "sum"):
```

Comment:
Thanks for the PR. Can you elaborate on why this change is necessary? AFAIK, Viewing the input from 3D to 2D is required for the CPP GEMM Template, but the CPP GEMM Template doesn't support in-place sum yet. Adding a corresponding test case to illustrate this would also be appreciated.

Reply:
hi @leslie-fang-intel appreciation for you explanation.  I just get the info that `sum` is not supported in CPP GEMM Template. I wrongly presume they have same ability previously. This PR is not necessary, I can adopt the dim collapsing logic in oneDNN integration for XPU instead. I will close this PR.

Thanks for your explanation again 👍 !